vizro-core/src/vizro/models/_components/form/_form_utils.py

```diff
@@ -69,7 +69,7 @@ def validate_max(cls, max, values):
     return max
 
 
-def validate_slider_value(cls, value, values):
+def validate_range_value(cls, value, values):
```

Comment:
We should probably update docstring below to something like:
"""Reusable validator for the "value" argument for range components."""

Let's also avoid using a word "slider/sliders" in the function implementation.